airflow/sensors/python.py

```diff
@@ -71,4 +71,6 @@ def poke(self, context: Context) -> PokeReturnValue | bool:
 
         self.log.info("Poking callable: %s", str(self.python_callable))
         return_value = self.python_callable(*self.op_args, **self.op_kwargs)
-        return PokeReturnValue(bool(return_value))
+        return (
+            return_value if isinstance(return_value, PokeReturnValue) else PokeReturnValue(bool(return_value))
```

Comment:
Because this line is so long should I update it to something like:
```
if isinstance(return_value, PokeReturnValue):
    return return_value
else:
    return PokeReturnValue(bool(return_value))
```

Reply:
I think splitting is a good idea, this is really quite long.